src/uwtools/config/formats/base.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import annotations
```

Comment:
This was causing a problem for Sphinx, as it led to type annotations being hidden from it at module-load time. The workaround, below, is to use the `UserDict` type instead of `Config`, which is correct in this case (the way the so-typed object is being used is appropriate for all `UserDict` instances, and instances of subclasses like `Config`).